app/src/main/res/layout/answer_layout.xml

```diff
@@ -1,65 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:orientation="vertical"
-    android:layout_marginTop="@dimen/activity_margin_vertical"
-    android:layout_marginBottom="@dimen/activity_margin_vertical">
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:orientation="vertical"
+  android:layout_marginTop="@dimen/activity_margin_vertical"
+  android:layout_marginBottom="@dimen/activity_margin_vertical">
 
     <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal">
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="horizontal"
+      android:background="#7AD9F970"
```

Comment:
hard coding colors is not a good practice, they should be defined in colors instead.